.kitchen.yml

```diff
@@ -1,8 +1,6 @@
 ---
 driver:
   name: vagrant
-  network:
-    - ["forwarded_port", {guest: 8983, host: 8888}]
```

Comment:
This gets applied to every vagrant instance kitchen starts and if multiple are run in parallel or if you leave one up while checking another it caused port conflicts in the host.